cloudinit/config/cc_seed_random.py

```diff
@@ -24,15 +24,19 @@
 optionally be specified in encoded form, with the encoding specified in
 ``encoding``.
 
+If the cloud provides its own random seed data, it will be appended to ``data``
+before it is written to ``file``.
+
 .. note::
     when using a multiline value for ``data`` or specifying binary data, be
     sure to follow yaml syntax and use the ``|`` and ``!binary`` yaml format
     specifiers when appropriate
 
-Instead of specifying a data string, a command can be run to generate/collect
-the data to be written. The command should be specified as a list of args in
-the ``command`` key. If a command is specified that cannot be run, no error
-will be reported unless ``command_required`` is set to true.
+A command can be specified (which will run after any data is written). The
```

Comment:
(the parens are not necessary here).  It also wasn't completely clear to me what or where "data was written".  That is possibly because I was just reading this as diff output, so maybe I lost context.

Maybe:

    If a ``command`` key is present, the command will be executed after ``file`` has been populated.
    The command's environment will contain the value of ``file`` specified above in the key 
    ``RANDOM_SEED_FILE``. If a command is specified that cannot be run, no error will 
    be reported unless ``command_required`` is set to true.